docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+---
+version: "3.8"
+
+services:
+  ollama:
+    build: .
+    container_name: ollama
+    image: ollama/ollama
+    healthcheck:
+      test: ollama --version || exit 1
```

Comment:
this test will always return true

Reply:
It won't return `True` if `ollama` is not in the container, so it check the executable is present and can be invoked.

I do agree that it's not the greatest health check over all. Changing this to use `curl`/`wget` or a Perl script is being discussed in [this thread](https://github.com/jmorganca/ollama/pull/1379#discussion_r1414501399)

Reply:
ollama is necessarily present if there's a build, curl would be better

Reply:
I am upstreaming this to the `Dockerfile` in https://github.com/jmorganca/ollama/pull/1909